### DISCUSSION TITLE
After hours of searching to no avail, I have found that you need to:

### DISCUSSION THREAD

**Discussion by patrick-amusivwa** [microsoft/WSL#9255] [General]
After hours of searching to no avail, I have found that you need to:

1. do `sudo su`  (all the commands need to be performed by root user)  ;
2.  delete the symlink in /tmp/.X11-unix  ( just do `rm /tmp/.X11-unix`) ; 
3.  run `vncserver` (It will now be able to create the server) ;
4.  Run `kex` and everything should work.

_Originally posted by @afonsofrancof in https://github.com/microsoft/WSL/discussions/6675#discussioncomment-1007781_

**Comment by patrick-amusivwa** [comment]
I'm still not able to delete any files or nine of the commands u mentioned non actually  worked on me 
![image](https://user-images.githubusercontent.com/108129819/205324966-9a5b800d-8164-481a-8efb-b72cfd6462eb.png)
![image](https://user-images.githubusercontent.com/108129819/205325020-381ec41d-c844-48b6-a2f1-b8d606008129.png)
kindly help with this

**Reply by afonsofrancof** [reply]
If it's a directory you could try 

rm -rf /tmp/.X11-unix

**Reply by patrick-amusivwa** [reply]
### i found a solution that worked 
for a reason [https://www.bleepingcomputer.com/news/security/kali-linux-gets-a-gui-desktop-in-windows-subsystem-for-linux/](url) worked for me i don't know why but it did .thanks @afonsofrancof  for the concern

![image](https://user-images.githubusercontent.com/108129819/205415419-99faab67-a249-4d00-bb79-e9313142e6c2.png)
thanks again!!
